client/src/helpers/ManageBookings/sortBookingsByStartDate.ts

```diff
@@ -0,0 +1,8 @@
+import { BookingRequests } from '../../pages/ManageBookings/types';
+
+export default function sortBookingsByStartDate(bookingRequests: BookingRequests): BookingRequests {
+  const sorted = [...bookingRequests].sort((bookingA, bookingB) => {
+    return bookingB.start.getDate() - bookingA.start.getDate();
+  });
+  return sorted;
+}
```

Comment:
This is a nice helper function, however, I would have loved to see it designed to be a bit more flexible as a bonus (e.g. an `asc/desc` distinction, maybe via a parameter, at the very least). Nice touch on making it a *pure function* though with the spread operator😎 💯